sale_credit_point/models/point_history.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models, api
+
+
+class PointHistory(models.Model):
+    _name = "credit.point.history"
+    _rec_name = "partner_id"
+
+    partner_id = fields.Many2one(
+        comodel_name="res.partner",
+        string="Partner",
+        required=True,
+    )
+    operation = fields.Selection(
+        string="Operation",
+        selection="_selection_operation",
+        required=True,
+    )
+    amount = fields.Monetary(
+        string='Amount',
+        currency_field='credit_point_currency_id',
+        readonly=True,
+        default=0,
+        required=True,
+    )
+    credit_point_currency_id = fields.Many2one(
+        related="partner_id.credit_point_currency_id",
+        readonly=True,
+    )
+    create_date = fields.Datetime()
+    comment = fields.Char(
+        string="Comment"
+    )
+
+    def _selection_operation(self):
```

Comment:
move this to a `POINT_OPERATIONS` var into partner.py then import it in the wizard and here